R/build-home-index.R

```diff
@@ -1,12 +1,17 @@
-build_home_index <- function(pkg = ".", quiet = TRUE) {
+build_home_index <- function(pkg = ".", quiet = TRUE, knit = TRUE) {
   pkg <- as_pkgdown(pkg)
 
   scoped_package_context(pkg$package, pkg$topic_index, pkg$article_index)
   scoped_file_context(depth = 0L)
 
+  # When knit = FALSE, prefer pre-rendered md file if exists
```

Comment:
Does this imply that if `knit = FALSE`, the Rmd exists, and the md doesn't, that the md will be built?

Reply:
In that case the Rmd will be rendered (but not knit) as if it was an md file.